lib/error/include/hse/error/merr.h

```diff
@@ -87,10 +85,7 @@ extern char hse_merr_bug2[];
 #define MERR_ERRNO_MASK (0x000000000000fffful)
 
 typedef hse_err_t merr_t;
-
-struct merr_info {
-    char buf[MERR_INFO_SZ];
-};
+typedef const char *(*merr_stringify)(int);
```

Comment:
Use modern notation:   typedef const char *merr_stringify(int);

Reply:
Didn't realize I was living in the past!